packages/core/src/components/utils/screen/mode.ts

```diff
@@ -7,22 +7,29 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-export type Mode = 'desktop' | 'mobile';
-const mobileMediaQuery = `only screen and (max-width: 767px)`;
+export type Mode = 'large' | 'medium' | 'small';
+
+let supportedModes: Mode[] = ['small', 'medium', 'large'];
+
+const smallMediaQuery = `only screen and (max-width: 480px)`;
+const mediumMediaQuery = `only screen and (min-width: 480px) and (max-width: 1024px)`;
```

Comment:
Should this be `481px`?

Reply:
Change `px` to `em`. Find assistant in this cheat sheet: https://gist.github.com/bartholomej/8415655